roles/network/defaults/main.yml

```diff
@@ -132,4 +132,5 @@ network_dummy_interface_mtu: 9000
 ##     dests:
 ##       - 10.10.0.2
 ##       - 10.10.0.3
+##     address: 192.168.1.1/24  # optional interface address
```

Comment:
I think we should rather makes this a list and iterate over it below, so we support v4+v6 or other cases with multiple IPs

Reply:
done